plugins/osdn/ovs/controller.go

```diff
@@ -294,6 +320,10 @@ func (c *FlowController) Setup(localSubnetCIDR, clusterNetworkCIDR, servicesNetw
 		return false, fmt.Errorf("Could not enable IPv4 forwarding on %s: %s", TUN, err)
 	}
 
+	// Table 253: rule version; note action is hex bytes separated by '.'
+	pluginVersion := getPluginVersion(c.multitenant)
+	otx.AddFlow("%s, %s%s.%s", VERSION_TABLE, VERSION_ACTION, pluginVersion[0], pluginVersion[1])
+
```

Comment:
Yeah, I did that last week and was testing it but apparently forgot to push it. Will do now.